.github/pull_request_template.md

```diff
@@ -1,35 +1,17 @@
-### Trello Card Link
-
-
 ### Description
+_What is the purpose of this PR? What is the current behavior? What is the new behavior? Is there anything else about this PR that we should know about?_
```

Comment:
Maybe add, are there any relevant links to this?

Reply:
like other relevant PR links?

Reply:
Yeah, or like in the current context, something like the link the Trello.